bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableIO.java

```diff
@@ -252,6 +263,11 @@ public boolean isCompletionMarker(Result[] result) {
     public long getRowCount(Result[] result) {
       return result == null ? 0 : result.length;
     }
+
+    @Override
+    public ByteString getKey(Result[] result) {
+      return isCompletionMarker(result) ? null : ByteString.copyFrom(result[result.length - 1].getRow());
```

Comment:
Is this code accessing the last element of the array without checking that the array is non-empty?